src/monitor/pgautofailover--1.5--1.6.sql

```diff
@@ -4,8 +4,66 @@
 -- complain if script is sourced in psql, rather than via CREATE EXTENSION
 \echo Use "CREATE EXTENSION pgautofailover" to load this file. \quit
 
-ALTER TABLE pgautofailover.node
-	RENAME TO node_upgrade_old;
+DROP FUNCTION
+     pgautofailover.register_node(text,text,int,name,text,bigint,int,int,
+                                  pgautofailover.replication_state,text,
+                                  int,bool,text);
+
+DROP FUNCTION
+     pgautofailover.node_active(text,int,int,
+                                pgautofailover.replication_state,bool,pg_lsn,text);
+
+DROP FUNCTION pgautofailover.get_other_nodes(int);
+
+DROP FUNCTION pgautofailover.get_other_nodes
+              (integer,pgautofailover.replication_state);
+
+DROP FUNCTION pgautofailover.current_state(text);
+DROP FUNCTION pgautofailover.current_state(text,int);
+
+DROP TRIGGER disable_secondary_check ON pgautofailover.formation;
+DROP FUNCTION pgautofailover.update_secondary_check() CASCADE;
```

Comment:
I think using CASCADE is pretty scary in a migration. It also doesn't seem to be necessary.
```suggestion
DROP FUNCTION pgautofailover.update_secondary_check();
```

Reply:
Good catch. I needed that while developing the script, and then realised it wasn't a good idea, as you're pointing out, and made it so that it's not needed anymore. Then forgot to actually remove the keyword. Oops. Done now.